src/methods/birth-date.js

```diff
@@ -25,8 +25,12 @@ const birthDate = customer => {
  * @returns {string}
  *
  * @example
- * // TODO
- *
+ * costumer will be the parameter of this function and as result will return his birth date
+ * var customer = object (body customer)
+ * customer can be like:
+ * const costumer = { "main_email": "joejonh@me.com","birth_date": {"year": 1990,"month": 10,"day": 1}}
```

Comment:
just making the code prettier:

```suggestion
 * const costumer = { main_email: 'joejonh@me.com', birth_date: { year: 1990, month: 10, day: 1 } }
```